lab/lab2/js/part2-app-state.js

```diff
@@ -34,10 +34,17 @@
 ===================== */
 
 // We set this to HTTP to prevent 'CORS' issues
-var downloadData = $.ajax("http://");
-var parseData = function() {};
-var makeMarkers = function() {};
-var plotMarkers = function() {};
+var downloadData = $.ajax("datasets/geojson/HousingCounselingAgencies.geojson");
```

Comment:
Where and what is this dataset?

Does it work for you? i get an error.

Reply:
This should work now.